### PR TITLE
Navbar - Added navbar styles

### PR DESCRIPTION
**Pull request by rsuescun**
## Summary
- Added navbar with logo and text
- Included all corrections from previous branches

## Screenshot
image

## Trello card
https://trello.com/c/962MC2E8/13-navbar

### CODE REVIEW

File: scss/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  display: flex;
+  flex-direction: row;
+  height: $nav-height;
+  justify-content: space-between;
+  padding: 20px;
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 100%;
+  z-index: 9999;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: why not 1 instead

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/styles.scss [hunk 2]
```diff
@@ -3,5 +3,5 @@
 }
 
 .padding-app {
-  padding: 20px;
+  padding: ($nav-height + 10) 20px 20px;
```
Comment (FrankIglesias) [inline scss/styles.scss]: 10px I think

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -10,6 +10,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body class="padding-app">
+  <header class="navbar">
+    <img class="navbar-logo" src="assets/wolox_logo.svg" alt="Logo App"/>
```
Comment (jmtov) [inline index.html]: Nice job naming these classes! 🤓

**Review by jmtov — APPROVED** [review]
Great job! 😺

**Review by FrankIglesias — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -10,6 +10,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body class="padding-app">
+  <header class="navbar">
```
Comment (mmamaniwolox) [inline index.html]: We usually rather using `<nav>` instead of `header`

File: scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
+$nav-height: 90px;
+$nav-logo-height: 30px;
+
+.navbar {
+  align-items: center;
+  background: $white;
+  border-bottom: 3px solid $grey;
+  display: flex;
+  flex-direction: row;
```
Comment (mmamaniwolox) [inline scss/navbar.scss]: This is default prop, you dont need to write it down

File: scss/navbar.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  height: $nav-height;
+  justify-content: space-between;
+  padding: 20px;
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+
+  .navbar-logo {
```
Comment (mmamaniwolox) [inline scss/navbar.scss]: Remember trying not to nest styles :P

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**